kolibri/plugins/coach/assets/src/views/recent-page/report-list.vue

```diff
@@ -54,8 +54,8 @@
       'content-icon': require('kolibri.coreVue.components.contentIcon'),
       'progress-bar': require('kolibri.coreVue.components.progressBar'),
     },
-    computed: {
-      reportList() {
+    watch: {
```

Comment:
I made a similiar change, but used `Array.from` and the built in `.sort` to avoid adding dependencies (credit: @rtibbles)

I'm not sure I agree with having a duplicate of the array in the view for the sake of it being sorted, particularly when this array is being defined in the `reports-list`'s `pageState`.

I had been sorting inside of the action before it was assigned to to the vuex state because it didn't seem like this data would be ordered in any other way inside of `report-list`, and having 2 of the same dataset seemed useless.